lib/onCreateWebpackConfig.ts

```diff
@@ -0,0 +1,51 @@
+import { CreateWebpackConfigArgs } from 'gatsby';
+
+// taken from https://github.com/staffe/gatsby-plugin-split-css/blob/master/gatsby-node.js
+/* MIT License
+ *
+ * Copyright (c) staffe
```

Comment:
TODO: Remove this after https://github.com/staffe/gatsby-plugin-split-css/pull/5